.github/workflows/buildAndTest.yaml

```diff
@@ -94,7 +94,10 @@ jobs:
           - metrics
           - kotlin
           - pubsub
```

Comment:
Do we still have the regular `it.pubsub=true` tests?

Reply:
Yes - for fairly basic sample integration tests